stan/math/prim/mat/meta/vector_seq_view.hpp

```diff
@@ -0,0 +1,85 @@
+#ifndef STAN_MATH_PRIM_MAT_META_VECTOR_SEQ_VIEW_HPP
+#define STAN_MATH_PRIM_MAT_META_VECTOR_SEQ_VIEW_HPP
+
+#include <stan/math/prim/mat/fun/Eigen.hpp>
+#include <vector>
+
+namespace stan {
+  using std::vector;
+  using Eigen::Matrix;
+  using Eigen::Dynamic;
+
+  /**
+   * Provides a low-cost wrapper for situations where you either need an Eigen
```

Comment:
I'd prefer complete sentences.  As in "This class provides ...".  And it should be just "std::vector" below --- that's the type.  You can wrap this up in code tags, but that's not a big deal.  I think "only allows std::vectors as the sequence type" is confusing given the name of the class---can you say "container type for sequence elements" instead?